meson.build

```diff
@@ -3080,7 +3080,7 @@ executable(
         link_with : [libshared],
         dependencies : [libaudit],
         install_rpath : rootlibexecdir,
-        install : true,
+        install : (conf.get('ENABLE_UTMP') == 1),
         install_dir : rootlibexecdir)
```

Comment:
Maybe, 
```
if conf.get('ENABLE_UTMP') == 1
        executable(...)
endif
```
is better.

Reply:
I did consider this first, but in #15601 some users might still want to copy over the file, or use it for tests. Seems safer to me that way

Reply:
Hmm, I cannot follow the discussion in #15601... But OK. Let's keep this in the current form.